zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/internal/client/HttpCall.java

```diff
@@ -13,17 +13,20 @@
  */
 package zipkin2.elasticsearch.internal.client;
 
-import java.io.Closeable;
+import com.fasterxml.jackson.databind.util.ByteBufferBackedInputStream;
```

Comment:
can we delay jackson to a later pull request with some of the okio utilities?

ex does one or the other here help?
https://github.com/square/okio/pull/346/files
https://github.com/square/okio/blob/d7fcee8db94177ac4d02e205f928d8425f352d52/okio/src/jvmTest/java/okio/NioTest.java

if not, add a TODO: to follow-up to swap out okio/moshi as we shouldn't really have multiple things doing the same if we can avoid it.

Reply:
Sure - fixed the bug in `ReadBuffer` and used it